vt100_output_windows.go

```diff
@@ -0,0 +1,299 @@
+// +build windows
```

Comment:
Wow, it's great library to support windows.
It seems to reduce the duplicate of code.